cli/src/maintenance.rs

```diff
@@ -403,6 +437,52 @@ impl SolidoState {
         Some((instruction, task))
     }
 
+    /// If there is a validator being deactivated, try to unstake its funds.
+    pub fn try_unstake_from_inactive_validator(&self) -> Option<(Instruction, MaintenanceOutput)> {
+        for validator in &self.solido.validators.entries {
+            // We are only interested in unstaking from inactive validators.
+            if validator.entry.active {
+                continue;
+            }
+            let try_unstake_balance = validator.entry.effective_stake_balance();
```

Comment:
We should unstake the amount in the first stake account, not the full balance; it might be different if there are multiple stake accounts.

Also, we need to check if the account is fully inactive, otherwise the transaction will fail. Fortunately we should have the balances already.

Reply:
Well spotted, I changed it in #409.
I think we should unstake all the stake account's balance.